packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -100,11 +103,19 @@ export default function ProfileButton({
                 }}
                 disableTooltip
               />
-              <ProfilePicture
-                user={user}
-                size={ProfileImageSize.Medium}
-                nativeLazyLoading
-              />
+              <span className="relative">
+                <ProfilePicture
+                  user={user}
+                  size={ProfileImageSize.Medium}
+                  nativeLazyLoading
+                />
+                {showProfileCompletionIndicator && (
+                  <InfoIcon
+                    size={IconSize.XSmall}
+                    className="absolute bottom-0 left-0 -translate-x-1/2 translate-y-1/2 rounded-full bg-background-subtle text-accent-cheese-default"
+                  />
+                )}
```

Comment:
Just recursion from above, but up to you if you wanna abstract it

Reply:
fixed